codespell_lib/data/dictionary.txt

```diff
@@ -20817,7 +20833,7 @@ menue->menu
 menues->menus
 menutitems->menuitems
 meny->menu, many,
-meory->memory
+meory->memory, maori,
```

Comment:
Can you capitalise `Maori`? It is only valid in aspell, hunspell, ispell, chrome browser, aoo-mozilla-en-dict when it is written in capitalise `Maori`. It is invalid when in lower-case.
```text
meory->memory, Maori,
```

Reply:
Done, [58178ac](https://github.com/codespell-project/codespell/pull/2445/commits/58178ac2efe45081ea20630046db5dd6084878be)